gallery/src/data-visualization/ReactVisBarGraphPage.tsx

```diff
@@ -0,0 +1,141 @@
+              </NxTableCell>
+            </NxTableRow>
+            <NxTableRow>
+              <NxTableCell>y</NxTableCell>
+              <NxTableCell>VerticalBarSeries: number <br /> HorizontalBarSeries: string/number</NxTableCell>
+              <NxTableCell>yes</NxTableCell>
+              <NxTableCell>
+                The y position in coordinates of the box to be used.
+              </NxTableCell>
+            </NxTableRow>
+          </NxTableBody>
+        </NxTable>
+      </section>
+      <section className="nx-tile-subsection">
+        <header className="nx-tile-subsection__header">
+          <h3 className="nx-h3">Basic API Reference</h3>
+        </header>
+        <NxTable>
+          <NxTableHead>
+            <NxTableRow>
+              <NxTableCell>Name</NxTableCell>
+              <NxTableCell>Type</NxTableCell>
+              <NxTableCell>Required</NxTableCell>
+              <NxTableCell>Description</NxTableCell>
+            </NxTableRow>
+          </NxTableHead>
+          <NxTableBody>
+            <NxTableRow>
+              <NxTableCell>barWidth</NxTableCell>
+              <NxTableCell>number/string/object</NxTableCell>
+              <NxTableCell>No</NxTableCell>
+              <NxTableCell>
+                The percentage for which each bar fills the designated bucket. 1.0 means that the bar fills the whole
+                bucket (no padding between bars), while a smaller percentage means more whitespace between the bars.
+              </NxTableCell>
+            </NxTableRow>
+            <NxTableRow>
+              <NxTableCell>color</NxTableCell>
+              <NxTableCell>string</NxTableCell>
+              <NxTableCell>No</NxTableCell>
+              <NxTableCell>
+                Assigns the specified color to the series.
+              </NxTableCell>
+            </NxTableRow>
+            <NxTableRow>
+              <NxTableCell>stroke</NxTableCell>
+              <NxTableCell>string</NxTableCell>
+              <NxTableCell>no</NxTableCell>
+              <NxTableCell>
+                In LineSeries, stroke is used instead of color to specify the color of the lines.
+              </NxTableCell>
+            </NxTableRow>
+            <NxTableRow>
+              <NxTableCell>tickFormat</NxTableCell>
+              <NxTableCell>function</NxTableCell>
+              <NxTableCell>No</NxTableCell>
+              <NxTableCell>
+                Format function for the tick label. Typically used to transform ticks to show addtional information.
+                For example, <code className="nx-code">{'tickFormat={val => `$${val}`}'}</code> adds a $ before
+                the tick label.
+              </NxTableCell>
+            </NxTableRow>
+            <NxTableRow>
+              <NxTableCell>title</NxTableCell>
+              <NxTableCell>string</NxTableCell>
+              <NxTableCell>No</NxTableCell>
+              <NxTableCell>
+                Shows the title for the axis.
```

Comment:
Which axis

Reply:
> Which axis

https://github.com/sonatype/sonatype-react-shared-components/pull/442/files#diff-01f9e2b8feee08c87c5551bc7fcfd37908269fed4267f1a52de73e7c54d8ceb0R112-R118

Reply:
> https://github.com/sonatype/sonatype-react-shared-components/pull/442/files#diff-01f9e2b8feee08c87c5551bc7fcfd37908269fed4267f1a52de73e7c54d8ceb0R112-R118

So this table is actually a combination of props for different components? I think that needs to be clearer.  Either a column needs to be added saying which component each prop goes on, or if you want to be significantly more thorough, you could create an entire tile subsection for each relevant component, with each having its own props table.